webpack/assets/javascripts/react_app/components/factCharts/FactChart.test.js

```diff
@@ -0,0 +1,134 @@
+import { shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import React from 'react';
+import FactChart from '.';
+import Store from '../../redux';
+
+import { getChartData, showModal, closeModal } from './FactChartActions';
+
+import { ajaxRequestAction } from '../../redux/actions/common/index';
+
+import reducer from './FactChartReducer';
+import * as types from './FactChartConstants';
+
+import {
+  initialState,
+  modalOpenState,
+  modalSuccessState,
+  modalLoadingState,
+  modalErrorState,
+  chartDataValues,
+} from './FactChart.fixtures';
+
+describe('factCharts', () => {
+  it('should render closed', () => {
+    const wrapper = shallow(
+      <FactChart data={{ id: 1, title: 'test title' }} store={Store} />
+    );
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  it('should render open', () => {
+    const wrapper = shallow(
+      <FactChart
+        data={{ id: 1, title: 'test title' }}
+        modalToDisplay
+        store={Store}
+      />
+    );
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  it('should open modal after click', () => {
+    const props = { onClick: jest.fn(), data: { id: 1 } };
+    const wrapper = shallow(<FactChart store={Store} {...props} />);
+
+    wrapper.simulate('click');
+    expect(props.onClick).toBeCalled();
+  });
+});
+
+jest.unmock('./FactChartActions');
+jest.mock('../../redux/actions/common');
+describe('factCharts actions', () => {
```

Comment:
can you move this to an `FactChartActions.test.js`? there is a helper method for testing actions `testActionsSnapshotWithFixtures` from `testHelpers`

Reply:
moved